### PR TITLE
Redesigned Sidebar

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Redesign Sidebar as per figma (resolves: #3781)

### Desktop Views

image

image

### Mobile View

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,7 +71,6 @@
     "libphonenumber-js": "^1.10.13",
     "lodash": "^4.17.21",
     "moment": "^2.29.4",
-    "postcss-loader": "^7.0.1",
```
Comment (rithviknishad) [inline package.json:73]: Removed as duplicate key error, see line 70.

Comment (khavinshankar) [reply package.json:73]: :+1:

**Comment by siddnikh** [comment]
Screenshot 2022-10-28 at 4 58 07 PM
I think it looks really nice, but the user info (here, Bodhish Thomas) seems a little off center. Also, how about repositioning the minimise icon? I think it might look better in some other place - just a suggestion.

**Comment by nihal467** [comment]
@rithviknishad the deployment preview is failed, check it

**Comment by nihal467** [comment]
@rithviknishad 
image

- [ ] the space between notice board and the notification is intentional

**Below Points are only for today's EOD call, if it get approved only, work on it**

- [ ] adding a split line feel good to me

image

- [ ] make the care icon and the footer user profile details little bit for bigger

Presentation1

**Comment by rithviknishad** [comment]
@nihal467 i've fixed the issue with spacing above notifications. ready for testing.
(refresh cache, or use incognito)

**Comment by nihal467** [comment]
test approved

**Pull request merged**